barretenberg/cpp/src/barretenberg/ultra_honk/ultra_honk.test.cpp

```diff
@@ -47,7 +47,12 @@ template <typename Flavor> class UltraHonkTests : public ::testing::Test {
     static void SetUpTestSuite() { bb::srs::init_crs_factory(bb::srs::get_ignition_crs_path()); }
 };
 
-using FlavorTypes = testing::Types<UltraFlavor, UltraZKFlavor, UltraKeccakFlavor, UltraKeccakZKFlavor>;
+using FlavorTypes = testing::Types<UltraFlavor,
+                                   UltraZKFlavor,
+                                   UltraKeccakFlavor,
+                                   UltraStarknetFlavor,
+                                   UltraKeccakZKFlavor,
+                                   UltraStarknetZKFlavor>;
```

Comment:
OK looking closer with a built copy
1. I dont think this test will work with `UltraStarknetZKFlavor` just plugged in
```
00:21:40 ultra_honk_tests: /home/aztec-dev/aztec-packages/barretenberg/cpp/src/barretenberg/ecc/fields/field_conversion.cpp:26: grumpkin::fr bb::field_conversion::convert_grumpkin_fr_from_bn254_frs(std::span<const bb::fr>): Assertion `(uint256_t(fr_vec[0]) < (uint256_t(1) << (NUM_LIMB_BITS * 2)))' failed.
00:21:40 barretenberg/cpp/scripts/run_test.sh: line 14: 1079001 Aborted                 (core dumped) ./bin/$1 --gtest_filter=$2
```
it fails our expected invariants